pkgs/by-name/ap/appflowy/package.nix

```
@@ -17,11 +17,11 @@ let
     rec {
       x86_64-linux = {
         urlSuffix = "linux-x86_64.tar.gz";
-        hash = "sha256-fn1UK8+7+vFL4nTnnRbfjCgttVYSw6pmmqabeHqTY3g=";
+        hash = "sha256-+8uHp4YkW/G2C618QPcd28zidvY3cLJuhZoSyIHRlDE=";
       };
       x86_64-darwin = {
         urlSuffix = "macos-universal.zip";
-        hash = "sha256-MoK6GlGmRVRp6feH8hab4CYpP4bXJN3XH7eHHSnhpS4=";
+        hash = "sha256-E4H6wLvb8ezxJaGxt9njvEjNY4jAJIx5agwVjqnTH1Y=";
```

Update to latest version
```suggestion
        hash = "sha256-lIxySROvdK8wCrBkN79ZMyVj3CWh3mGenuz6UDGDQf4=";
```